### PR TITLE
Add armv71 config

### PR DESCRIPTION
**Pull request by filiptc**
Fixes TooTallNate/node-vorbis#21

### CODE REVIEW

File: deps/libvorbis/config/linux/armv71/config.h [hunk 1]
```diff
@@ -0,0 +1 @@
+<html><body>You are being <a href="https://raw.githubusercontent.com/TooTallNate/node-vorbis/master/deps/libvorbis/config/linux/x64/config.h">redirected</a>.</body></html>
```
Comment (TooTallNate) [inline deps/libvorbis/config/linux/armv71/config.h]: This doesn't seem correct 🤔

**Review by TooTallNate — CHANGES_REQUESTED** [review]

File: deps/libvorbis/config/linux/armv71/config.h (hunk 1 above)
Comment (filiptc) [reply deps/libvorbis/config/linux/armv71/config.h]: Oh gosh. :man_facepalming: Let me fix that real quick.

Comment (filiptc) [reply deps/libvorbis/config/linux/armv71/config.h]: Curl and being in a rush will bite you.

**Comment by TooTallNate** [comment]
Thank you!

**Pull request merged**